crypto/mod.ts

```diff
@@ -133,6 +134,7 @@ const stdCrypto = ((x) => x)({
         return fnv(algorithm.name, bytes);
       }
 
+      const wasmCrypto = instantiateWasm();
```

Comment:
This caches the result of previous runs, so calling this multiple times is perfectly fine and will be very fast the second time.